src/status_im/ui2/components/toolbar.cljs

```diff
@@ -0,0 +1,40 @@
+(ns status-im.ui2.components.toolbar
```

Comment:
why are we moving old components ?

Reply:
so there are two options . 1 - you just use old component for now, and later it will be reworked (so no changes in old components) or 2 - you create component from scratch in `2` namespace with new designs and components from quo2